install/0000_80_machine-config-operator_02_images.configmap.yaml

```diff
@@ -7,11 +7,7 @@ data:
   images.json: >
     {
       "machineConfigOperator": "registry.svc.ci.openshift.org/openshift:machine-config-operator",
-      "machineConfigController": "registry.svc.ci.openshift.org/openshift:machine-config-controller",
-      "machineConfigDaemon": "registry.svc.ci.openshift.org/openshift:machine-config-daemon",
-      "machineConfigServer": "registry.svc.ci.openshift.org/openshift:machine-config-server",
       "etcd": "registry.svc.ci.openshift.org/openshift:etcd",
-      "setupEtcdEnv": "registry.svc.ci.openshift.org/openshift:setup-etcd-environment",
```

Comment:
I think i might have to revert this setupEtcdEnv deletion for the .yaml below.  Will wait for ci..

Reply:
uhm, which yaml?